From 38dd95605796c83e9fee6d19cad5621af6a392d3 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Mon, 12 Nov 2018 09:17:51 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808: Add usb_pcie_grf to pcie0 node

The PCIe driver need configure some bits after passing link
trainning, which belongs to the controller driver part but
can not make use of PHY API.

Change-Id: I8c3d5f8a25de6184bee572c7c954c9f3d9df20c2
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index 75d889c592f7..90dea14e76de 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -179,6 +179,7 @@
 			      "ctl-core", "ctl-nsticky", "ctl-sticky",
 			      "ctl-pwr", "ctl-niu-a", "ctl-niu-p",
 			      "phy-por", "phy-p", "phy-pipe";
+		rockchip,usbpciegrf = <&usb_pcie_grf>;
 		status = "disabled";
 	};
 
-- 
2.35.3

